settings.txt

```diff
@@ -808,9 +808,9 @@ Transient errors in this log can be ignored unless you are having issues=Vorübe
 Unraid API extra origins=Unraid-API Extra Origins
 View Flash Backup Error Log=Flash-Backup-Fehlerprotokoll anzeigen
 
-; Note: April 29, 2023 - 5 missing translations
+; Note: April 28, 2023 - 3 missing translations
 Both Web Virt-Manager Remote Viewer=Both Web & Virt-Manager Remote Viewer
-Console Options=
-Virt-Manager Remote Viewer=
-Web interface=
-Remote T2FA requires Unraid 610 or higher=Remote T2FA requires Unraid 6.10 or higher
+Console Options=Konsolen-Optionen
+Virt-Manager Remote Viewer=Fernansicht der Virt-Verwaltung
+Web interface=Webinterface
+Remote T2FA requires Unraid 610 or higher=Entfernte T2FA erfordert Unraid 6.10 oder höher
```

Comment:
Was sagst du in diesem Fall zu `Remote` statt `Entfernte`? Es ist mir schon klar das du aus der Ferne zugreifst aber Remote sollte von den meisten Nutzern verstanden werden? Was ist deine Meinung hierzu @kennymc-c ?

Reply:
Ich hab mich da an die bisherige Übersetzung von Remote z.B. bei Fernzugriff gehalten. Bezieht sich das hier auch darauf? Dann könnte man z.B. `T2FA für Fernzugriff` nehmen.

Reply:
T2FA = Trusted Two Factor Authentication
`T2FA für Fernzugriff erforder Unraid 6.10 oder höher` passt dann in dem Kontext perfekt ja.